geopandas/tests/test_vectorized_api.py

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
I added this as I was connecting GeometryArray to GeoSeries.  Yes, it could probably be merged elsewhere.